### PR TITLE
Add setters and getters for imageLoadFunction

### PR DESCRIPTION
**Pull request by bartvde**
We have the same for tileLoadFunction so adding here as well makes sense to me

### CODE REVIEW

File: src/ol/source/imagemapguidesource.js [hunk 1]
```diff
@@ -161,6 +161,15 @@ ol.source.ImageMapGuide.prototype.getImage =
 
 
 /**
+ * @return {ol.ImageLoadFunctionType} ImageLoadFunction
```
Comment (elemoine) [inline src/ol/source/imagemapguidesource.js]: "ImageLoadFunction" -> "The image load function".

**Comment by elemoine** [comment]
I just added a doc-related comment. Otherwise looks good to me.

File: src/ol/source/imagemapguidesource.js (hunk 1 above)
Comment (bartvde) [inline src/ol/source/imagemapguidesource.js]: Copy / paste from here :-) https://github.com/openlayers/ol3/blob/master/src/ol/source/tileimagesource.js#L128
I'll adapt

**Comment by bartvde** [comment]
thanks @elemoine but not merge as yet until you have released right?

**Comment by elemoine** [comment]
> thanks @elemoine but not merge as yet until you have released right?

Others did not care :-) Please merge.

**Pull request merged**